react/features/base/config/functions.js

```diff
@@ -43,6 +43,21 @@ const WHITELISTED_KEYS = [
      */
     'callDisplayName',
 
+    /**
+     * The call handle of the CallKit call representing the conference/meeting
+     * associated with this config.js. The property is meant for use cases in
+     * which Jitsi Meet is to work with a CallKit call created outside of Jitsi
+     * Meet and to be adopted by Jitsi Meet such as, for example, an incoming
+     * and/or outgoing CallKit call created by Jitsi Meet SDK for iOS
+     * clients/consumers prior to giving control to Jitsi Meet. As the value is
```

Comment:
Again, the comment is inaccurate! "The property is meant for use cases in which Jitsi Meet is to work with a CallKit call created outside of Jitsi Meet and to be adopted by Jitsi Meet such as, for example, an incoming and/or outgoing CallKit call created by Jitsi Meet SDK for iOS clients/consumers prior to giving control to Jitsi Meet." I'll take care of this one during the merge